config/initializers/extends.rb

```diff
@@ -222,7 +222,9 @@ class Extends
     edit_image_on_result: 110,
     edit_image_on_step: 111,
     edit_image_on_step_in_repository: 112,
-  }
+    share_inventory: 113,
+    unshare_inventory: 114
```

Comment:
You should also add it to corresponding `ACTIVITY_GROUPS` below, probably `inventories` or `team`?